praetorian_cli/sdk/test/ui/test_aegis_schedule.py

```
@@ -0,0 +1,552 @@
+
+# --- Mock SDK with schedules support ---
+
+class MockSchedulesAPI:
+    """Mock for sdk.schedules that records calls."""
+
+    def __init__(self, schedules=None, error=None):
+        self._schedules = schedules or []
+        self._error = error
+        self.calls = []
+
+    def list(self, pages=1):
```

Ruff (0.14.14)

[warning] Unused method argument: `pages`

(ARG002)